site/content/docs/monitoring/false-positives.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Managing false positives
+weight: 4
+menu:
+  platform:
+    parent: "Monitoring"
+---
+
+Effective monitoring stands and falls with its accuracy. You want to be alerted-and, if it comes to it, woken up in the middle of the night-only when there's a real issue. However, the reality is that false positives can occur for various reasons. A check might be affected by a harmless glitch in the network connection between Checkly and your check target, causing your site to be flagged as slow or unreachable, when, in fact, it is running perfectly fine.
+
+The public internet, with its millions of networks and nodes, can be unpredictable, making occasional hiccups unavoidable. Thankfully, many of these are just isolated incidents that resolve on their own, and there are ways to protect your monitoring setup from them.
+
+In this section, we'll dive into why these issues happen and share strategies on minimizing the impact of false positives, allowing you to more clearly distinguish between a real problem and a simple network blip.
+
+## Common Causes of Performance Variations
+
+When setting up your monitoring strategy, it's important to consider a few common factors that can affect networking performance:
+
+- **Network Latency:** The time it takes for data to travel between points. This can vary due to distance, routing, or congestion.
+- **Packet Loss:** Sometimes data packets don’t reach their destination and need to be resent, leading to delays.
+- **Server Response Time:** Servers might occasionally take longer to respond, or load balancers and backend services could be at capacity.
+- **Infrastructure Issues:** Temporary issues like routing errors, server load spikes, or scheduled maintenance can also impact performance.
```

Comment:
should we mention security here too? "firewall blockings can occur..." or similar?